biodata-models/src/main/java/org/opencb/biodata/models/clinical/qc/Signature.java

```diff
@@ -51,13 +51,19 @@ public class Signature {
             description = FieldConstants.SIGNATURE_FILES_DESCRIPTION)
     private List<String> files;
 
-    @DataField(id = "fitting", indexed = true,
-            description = FieldConstants.SIGNATURE_SIGNATURE_FITTING_DESCRIPTION)
+    @Deprecated
+    @DataField(id = "fittingScore", indexed = true,
```

Comment:
Isn't the id supposed to be the old `fitting`?

Reply:
You're right.